internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+package message
+
+import (
+	"github.com/tomarrell/lbadd/internal/compiler/command"
+)
+
+// ConvertCommandToMessage converts a command.Command to a message.Command
+func ConvertCommandToMessage(cmd command.Command) Message {
+	switch c := cmd.(type) {
+	case *command.Scan:
```

Comment:
Do we use pointers in the compiler? I don't think we use pointers to these structs ever. Use the actual values please. To ensure this works, add tests that use actual compiler output, to ensure this works and is usable.

Reply:
`*command.Scan` implements `command.Command` and not `command.Scan` thus.

Ref: https://github.com/tomarrell/lbadd/blob/master/internal/compiler/command/command.go#L12

Reply:
No, that's a compiler check
`command.Scan` implements `command.Command`
https://github.com/tomarrell/lbadd/blob/master/internal/compiler/command/command.go#L320